public/js/save.js

```diff
@@ -1,3 +1,42 @@
+/*
+* Oauth with google logins.
+*/
+
+// Run when signed in.
+function onSignIn(googleUser) {
+    console.log("User logged in.");
+    var id_token = googleUser.getAuthResponse().id_token;
+    var xhr = new XMLHttpRequest();
+    xhr.open('POST', '/tokensignin');
```

Comment:
I'll try using ajax. In my rush to wanting to get it working, I just went quickly and used code examples. Thanks!